src/solver/conv_hip_implicit_gemm_bwd_v4r1.cpp

```diff
@@ -749,6 +749,8 @@ bool ConvHipImplicitGemmBwdDataV4R1::IsApplicable(const ConvolutionContext& ctx)
 
     if(miopen::IsDisabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_BWD_V4R1{}))
         return false;
+    if(miopen::IsEnabled(MIOPEN_DEBUG_CONVOLUTION_DETERMINISTIC{}))
+        return false;
```

Comment:
I think hip igemm v4r1 is deterministic. Why do we ban this solver?

Reply:
For debug purpose(not consider performance), need to disable a solver if any of the configs may use atomic. So I just disable all suspicious one.